test_config.yaml

```
@@ -67,10 +67,14 @@ test_server:
     # from the real server.
     /node_agent2/node_agent: http://161.72.57.3:8080 # LT
     /api/requestgroups/: https://observe.lco.global # LCO
+    /api/triggers/ztf: http://skipper.caltech.edu:4000 # ZTF
 
 app:
   lt_host: localhost
   lt_port: 64502
   lco_protocol: http
   lco_host: localhost
   lco_port: 64502
+  ztf_protocol: http
+  ztf_host: localhost
+  ztf_port: 64502
```

```suggestion
  ztf:
    protocol: http
    host: localhost
    port: 64502
```
I think this is why it's not triggering the test server